python-package/lightgbm/basic.py

```diff
@@ -1245,12 +1245,12 @@ def __getstate__(self):
         return this
 
     def __setstate__(self, state):
-        model_str = state.get('handle', None)
+        model_str = state.get('handle', None)  # list of str
```

Comment:
It's already replaced. You can print type(model_str) to see it.

I am sure it's not a bug. It's exactly same logic with dumped model. Otherwise, model copy, feature importance and plotting library won't pass the travis check.